src/main/kotlin/fi/metatavu/timebank/api/controllers/VacationRequestController.kt

```diff
@@ -37,6 +37,7 @@ class VacationRequestController {
                 message = vacationRequest.message,
                 createdAt = vacationRequest.createdAt,
                 updatedAt = vacationRequest.updatedAt,
+                draft = vacationRequest.draft
```

Comment:
According to the issue this PR is resolving, nothing related to `VacationRequests` should've changed in this PR?

It seems to me that this branch has originally been based on the earlier PR but that base branch had some more work done before merging.
Therefore I recommend running `git fetch` > `git merge origin/develop` and resolving the conflicts in your IDE. As you can see in this PR, Github is warning you that it has conflicts with the `develop` branch.